sda_download/sda_download.go

```diff
@@ -44,8 +44,9 @@ var ArgHelp = `
 	[uri]
 		All flagless arguments will be used as sda-download uri.
 	[filepath(s)]
-		The filepath of the file to download. If no filepath is provided
-        then the whole dataset will be downloaded.
+		The filepath of the file to download.
+    [fileid(s)]
```

Comment:
Fix the indentation here

Reply:
dirpath below is also off

Reply:
strange, I do not see them off locally

Reply:
I fixed it on spot because in my case they were aligned